src/App.tsx

```diff
@@ -22,6 +22,10 @@ export class App extends React.Component< {}, State > {
   );
 
   render() {
+    if (this.state.pressedKey === ' ') {
```

Comment:
better use `event.code` to check if the key is `space`
and don't forget to deploy your changes 😉 
<img width="198" alt="image" src="https://user-images.githubusercontent.com/55802630/183098105-c18858ba-a346-4062-b9e2-b7e616c25678.png">